### ISSUE TITLE
ons-tabbar.ts: target.element must be an instance of Element.

### ISSUE THREAD

**Issue by asial-matagawa** [OnsenUI/OnsenUI#1728]
__Environment__ 

```
[Core]
    Onsen UI 2.0.4

[Framework]
    Angular 2.2.4

[Framework binding]
    angular2-onsenui 1.0.0-rc.3

[Additional libraries]
    None

[Platform]
    Desktop - Windows 10

[Browser]
    Desktop - Chrome 54.0.2840.99
```

__Encountered problem__
PageLoader call in `ons-tabbar.ts` (OnsTab Directive) does not work properly.

![snapcrab_noname_2016-12-4_4-56-26_no-00](https://cloud.githubusercontent.com/assets/19771915/20861871/0a55b20e-b9de-11e6-8f2b-bef024c4d740.png)

Note:
- angular2-onsenui 1.0.0-rc.3 causes this problem. The previous versions work fine.

__How to reproduce__
```js
@Component({
    selector: 'app',
    template: `
        <ons-tabbar>
            <ons-tab label="Home1" active>
            </ons-tab>
        </ons-tabbar>
    `,
    providers: []
})
class AppComponent implements AfterViewInit {
    @ViewChildren(OnsTab) tabQueryList: QueryList<OnsTab>;

    ngAfterViewInit() {
        this.tabQueryList.toArray()[0].page = TwitterPageComponent;
    }
}
```

Here is a Plunker which reproduces this problem. See `src/main.ts`.
https://plnkr.co/edit/JYQAo6yaGueJnfW4V81W

**Comment by frankdiox** [comment]
@asial-matagawa Could you try with master branch? I made a refactor of `pageLoader` that hasn't been released yet #1671

**Comment by asial-matagawa** [comment]
@frankdiox 
I will try it soon. Thank you for information!

**Comment by frankdiox** [comment]
@asial-matagawa I made the refactor 4 weeks ago and it included changes to the core and also angular2 bindings. looks like `angular2-onsenui@1.0.0-rc.3` was released 3 weeks ago but the core is still the same. I think this must be the problem. #1727 could also be related.

**Comment by asial-matagawa** [comment]
@frankdiox 
I tried with the latest revision dc9f004a007c94977c8717d70c5fdd02e2a5fbf8 and then this problem disappeared.
(Instead, #1727 occured. It seems that #1727 has nothing to do with this issue)

Let's release Onsen UI 2.0.5 and angular2-onsenui 1.0.0-rc.4.

**Comment by asial-matagawa** [comment]
This issue will be solved in the next version of Onsen UI (=2.0.5) and angular2-onsenui (=1.0.0-rc.4).

Closing this.
